discovery/gossiper.go

```diff
@@ -1211,6 +1218,14 @@ func (d *AuthenticatedGossiper) retransmitStaleAnns(now time.Time) error {
 		// introduction of the MaxHTLC field, then we'll update this
 		// edge to propagate this information in the network.
 		if !edge.MessageFlags.HasMaxHtlc() {
+			// We'll make sure we support the new max_htlc field if
```

Comment:
maybe we should move this check/update to the local manager instead? Making gossiper as unaware of local changes as possible,

Reply:
I wanted to keep the change set small. Moving retransmission of stale channels to the local chan mgr is a much larger change. Also not sure if that is the place where it belongs.